docker/graphite-metrictank.yaml

```diff
@@ -1,18 +1,11 @@
 cache:
-  CACHE_DIR: /tmp/graphite-api-cache
-  CACHE_TYPE: filesystem
+  dir: /tmp/graphite-api-cache
```

Comment:
Using caching could cause problems for you.  The caching layer in graphite is not multi-tenant and so if you make a query with on orgId, then make the same query for a different orgId, you will get the cached data back